src/plone/restapi/services/controlpanels/configure.zcml

```diff
@@ -2,7 +2,15 @@
     xmlns="http://namespaces.zope.org/zope"
     xmlns:plone="http://namespaces.plone.org/plone">
 
-  <include package="plone.app.controlpanel" file="permissions.zcml" />
+  <include
+    zcml:condition="have plone-52"
```

Comment:
maybe use `zcml:condition="not-installed plone.app.controlpanel"`?

Reply:
Indeed, better make them exclusive, if we don't we might have use cases when both are true.